src/app/templates/client/Client.jsx

```diff
@@ -123,10 +148,14 @@ function Client() {
       onDragLeave={handleDragLeave}
       onDrop={handleDrop}
     >
-      <div className="navigation__wrapper">
+      <div className={`navigation__wrapper
+        ${activeView !== viewPossibilities.nav ? 'non-focused-ui' : ''}`}
```

Comment:
`non-focused-ui` classname doesn't follow BEM methodology and can cuz class name collision.